### PR TITLE
kv: track and output cumulative latency

### PR DESCRIPTION
**Pull request by petermattis**
This change is [Reviewable](https://reviewable.io/reviews/cockroachdb/loadgen/45)

### CODE REVIEW

File: kv/main.go [hunk 1]
```diff
@@ -585,6 +587,7 @@ func main() {
 				}
 			}
 
+			cumLatency.Merge(h)
```
Comment (rjnn) [inline kv/main.go:590]: Merge returns a value (number of values "dropped"). Do you know what that is and if we should care?

**Review by rjnn — APPROVED** [review]
LGTM

File: kv/main.go (hunk 1 above)
Comment (petermattis) [reply kv/main.go:590]: It returns the number of samples that could not be merged. I believe that can only happen if you merge 2 histograms that were configured differently. Our histograms are all configured identically.

Comment (rjnn) [reply kv/main.go:590]: Great. I could not figure that out from the docs. Thanks.

Comment (petermattis) [reply kv/main.go:590]: Docs lie. Read the source.

**Pull request merged**